shared/utility/skill/scripts/docker/Relax.yaml

```diff
@@ -1,28 +1,82 @@
-[
-  {
-    "duration": 25,
-    "targets":
-      [
-        { "id": "R_SHOULDER_PITCH", "position": 0.7, "gain": .NaN, "torque": 0 },
-        { "id": "L_SHOULDER_PITCH", "position": 0.7, "gain": .NaN, "torque": 0 },
-        { "id": "R_SHOULDER_ROLL", "position": 0.7, "gain": .NaN, "torque": 0 },
-        { "id": "L_SHOULDER_ROLL", "position": 0.7, "gain": .NaN, "torque": 0 },
-        { "id": "R_ELBOW", "position": -1.5, "gain": .NaN, "torque": 0 },
-        { "id": "L_ELBOW", "position": -1.5, "gain": .NaN, "torque": 0 },
-        { "id": "R_HIP_YAW", "position": 0, "gain": .NaN, "torque": 0 },
-        { "id": "L_HIP_YAW", "position": 0, "gain": .NaN, "torque": 0 },
-        { "id": "R_HIP_PITCH", "position": 0, "gain": .NaN, "torque": 0 },
-        { "id": "L_HIP_PITCH", "position": 0, "gain": .NaN, "torque": 0 },
-        { "id": "R_KNEE", "position": 0, "gain": .NaN, "torque": 0 },
-        { "id": "L_KNEE", "position": 0, "gain": .NaN, "torque": 0 },
-        { "id": "R_ANKLE_PITCH", "position": 0, "gain": .NaN, "torque": 0 },
-        { "id": "L_ANKLE_PITCH", "position": 0, "gain": .NaN, "torque": 0 },
-        { "id": "R_HIP_ROLL", "position": 0, "gain": .NaN, "torque": 0 },
-        { "id": "L_HIP_ROLL", "position": 0, "gain": .NaN, "torque": 0 },
-        { "id": "HEAD_YAW", "position": 0, "gain": .NaN, "torque": 0 },
-        { "id": "HEAD_PITCH", "position": 0, "gain": .NaN, "torque": 0 },
-        { "id": "R_ANKLE_ROLL", "position": 0, "gain": .NaN, "torque": 0 },
-        { "id": "L_ANKLE_ROLL", "position": 0, "gain": .NaN, "torque": 0 },
-      ],
-  },
-]
+- duration: 25
+  targets:
+    - id: R_SHOULDER_PITCH
+      position: 0
+      gain: .NaN
```

Comment:
Is the gain supposed to be NaN here? I would have thought it should be 0 too? Or is this a divide by zero thing?

Reply:
pretty sure it's what it needs to be to relax the servos - not sure if 0 does the same thing but I can test